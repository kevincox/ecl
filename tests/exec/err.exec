cmd: load err.ecl -flines
status: 1
