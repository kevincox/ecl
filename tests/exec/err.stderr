1.13 Invalid reference "fart"

Error occured.
